Change exception to use stack trace in android (#741)
microsoft/appcenter-sdk-react-native: CHANGELOG.md

  - Accounts in any Azure AD directory
  - Accounts in any Azure AD directory and personal Microsoft accounts

### App Center Crashes

#### Android

> * **[Behavior Change]** Fixes a security issue in the way native Android crashes are processed. The exception message will now be `null` and the exception stack trace might look slightly different.

Reviewer: ```suggestion

* **[Behavior Change]** Fix a security issue in the way native Android crashes are processed. The exception message is now `null` and the exception stack trace is the raw Java stack trace.
```
